assets/i18n/en.json

```diff
@@ -37,6 +37,7 @@
   "I18N_ERROR_PAGE_TITLE_404": "Error 404 - Oppia",
   "I18N_ERROR_PAGE_TITLE_500": "Error 500 - Oppia",
   "I18N_FOOTER_ABOUT": "About",
+  "I18N_FOOTER_GET_STARTED": "Get Started",
```

Comment:
No, there are still two I18N_FOOTER_GET_STARTED constants in en.json, on lines 40 and 49. See

https://github.com/oppia/oppia/pull/2712/files